source/agora/common/Block.d

```diff
@@ -217,6 +217,26 @@ public struct Block
         }
         return hash;
     }
+
+    /*******************************************************************************
+
+        Find the sequence of transactions for the hash.
+
+        Params:
+            hash = `Hash` of `Transaction`
+
+        Returns:
+            Return sequence of transactions for the hash
```

Comment:
It should document what it returns if the hash index is not found.

Reply:
Okay~